### PR TITLE
v2

### PR DESCRIPTION
**Pull request by jacobgunnarsson**
- Removed the old media query mixins in favor of easier `above`, `below` and `between` mixins. 
- Set three new breakpoints available in a `$breakpoints` hash.
- Removed bower.json
- Changed the BEM style variable namespacing syntax to just a single dash, i.e. `$namespace--var` `$namespace-var`
- Added easing variables after the most commonly named easing functions (quad, cubic, quart and so on)
- Removed typography mixins and set default typography `html` properties in `_base.scss`
- Removed all compiled utility classes in `_utils.scss`
- Added `rem` helper function 
- Bumped version number to `2.0.0` and bumped package deps.
- Fixed deprecated stylelint rules

### CODE REVIEW

**Comment by eriklindebratt** [comment]
Can we get rid of the easing variables? I personally never use them anyway.

File: README.md [hunk 1]
```diff
@@ -73,19 +63,20 @@ vendor
...
-We use `stylelint` to ensure a consistent style. See `/src/helpers/_styleguide-example.scss`
-for an example file that adheres to the `/.stylelintrc` rules. To lint manually run
-`npm run lint`, this task is also run before each commit as a pre-commit hook.
+We use `stylelint` to ensure a consistent style. See
+`/src/helpers/_stylelint-example.scss` for an example file that adheres to the
+`/.stylelintrc` rules. To lint manually run `npm run lint`, this task is also
+run before each commit as a pre-commit hook.
 
 ## Gulp
 To compile the SCSS run `npm install && gulp`. This is handy to ensure you don't
```
Comment (eriklindebratt) [inline README.md]: Change to `npm install && node_modules/.bin/gulp` and/or add `npm` script `dev`?

File: src/helpers/_mixins.scss [hunk 2]
```diff
@@ -2,9 +2,6 @@
  * _mixins.scss
  */
 
-/*
- * General mixins
- */
 @mixin clearfix { overflow: hidden; }
```
Comment (eriklindebratt) [inline src/helpers/_mixins.scss]: IMO we should get rid of this, or change it to be what `clearfix-overflow-visible` does.

Comment (jacobgunnarsson) [reply src/helpers/_mixins.scss]: I agree, we should remove the clearfixes altogether.

File: README.md (hunk 1 above)
Comment (jacobgunnarsson) [reply README.md]: Changed to `npm install && node_modules/.bin/gulp`

**Comment by jacobgunnarsson** [comment]
@eriklindebratt  Regarding easing variables, maybe we should break out variable types into separate partials. I.e. we have a bucket export in `base/variables/_index.scss`, that imports from `base/variables/_easings.scss`, `base/variables/_colors.scss` or `base/variables/_breakpoints.scss` and so on.

I would still like the easing variables, but would like to make it easier to read/manage variables. I've found that the variables partial has been almost too big to manage in some projects.

**Pull request merged**